app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class User < ActiveRecord::Base
+  before_save { self.email = email.downcase }
+  validates :name, presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
This seem like a good thing to put in a helper class. Does rails have ``is_valid_email?`` or something